eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -558,6 +566,7 @@ object Router {
   case object GetChannels
   case object GetChannelsMap
   case object GetChannelUpdates
+  case object PrintChannelUpdates
```

Comment:
This was probably for your internal tests, remove?

Reply:
I thought it could come in handy in the future, but now that we have [8aee27a](https://github.com/ACINQ/eclair/pull/2224/commits/8aee27a724cfef6f2b269e7ab37544b5dff77651) we can probably get rid of it. Let me append that commit to this PR.

Reply:
I think this is the kind of code you keep in a local branch to help debug stuff, but we shouldn't have it on `master`.